src/main/scala/com/fulcrumgenomics/util/Metric.scala

```diff
@@ -103,19 +103,45 @@ object Metric {
     * line should be a single metric. */
   def read[T <: Metric](path: Path)(implicit tt: ru.TypeTag[T]): Seq[T] = read[T](Io.readLines(path), Some(path.toString))
 
+  /** Writes one or more metrics to the given writer.  The first line will be a header with the field names.  Each subsequent
+    * line is a single metric.
+    */
+  def write[T <: Metric](writer: Writer, metric: T*)(implicit tt: ru.TypeTag[T]): Unit = {
+    val out = new MetricWriter[T](writer)
+    out.write(metric:_*)
+    out.close()
+  }
+
+  /** Writes one or more metrics to the given path.  The first line will be a header with the field names.  Each subsequent
+    * line is a single metric.
+    */
+  def write[T <: Metric](path: Path, metric: T*)(implicit tt: ru.TypeTag[T]): Unit = write(Io.toWriter(path), metric:_*)
+
+  /** Writes a metric to the given writer.  The first line will be a header with the field names.  Each subsequent
+    * line is a single metric.
+    */
+  def write[T <: Metric](writer: Writer, metrics: TraversableOnce[T])(implicit tt: ru.TypeTag[T]): Unit = write(writer, metrics.toSeq:_*)
```

Comment:
Good point.  I'll fix this but note that I am not fixing up metric reading, which also uses `Seq` and doesn't have a `MetricReader`.